opencti-platform/opencti-front/src/private/components/workspaces/investigations/InvestigationExpandForm.tsx

```diff
@@ -348,9 +348,12 @@ const InvestigationExpandFormContent = ({
       .filter(({ value }) => value > 0)
       .sort((a, b) => a.label.localeCompare(b.label));
 
+    // Uses to determine which key of translation to use below.
+    const relNames = ['Part-Of', 'Located-At', 'Indicates', 'Mitigates', 'Targets', 'Uses', 'Related-To', 'Attributed-To'];
```

Comment:
Open to suggestions if you have better idea to check in mind

Reply:
I'm not sure to understand how we can see relationships in the targets list. 
The dialog displays targets on the left and relationships on the right, how can we have relationships on the left part ? @lndrtrbn

Reply:
Because we can have relations targeting other relations

Reply:
is this list complete ? I can see we have a lot more in STIX_CORE_RELATIONSHIPS

Reply:
could you get it from the schema ? 

```ts
const { schema } = useAuth();
// ...
const relationshipsNames = schema.sros.map(({id, label }) => label)
```
Something like that I believe

Reply:
see how it's done is `useSearchEntities`, we use the schema to get generate all possible options in the filter autocomplete

Reply:
Ok I'll have a look thanks @labo-flg

Reply:
Indeed, seems way better now